.github/workflows/handle-new-external-pr.yml

```diff
@@ -28,6 +29,11 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install pipenv==2021.5.29
+
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v14.6  # disable-secrets-detection
```

Comment:
What is the meaning of the `# disable-secrets-detection` - why do we need that?

Reply:
not sure, perhaps i copied it by mistake, removing it.